src/core/index.ts

```diff
@@ -14,6 +14,13 @@ import { ValueError, PermissionError, KeyError } from '../errors';
 import { Codec } from "../compression/types";
 import { getCodec } from "../compression/creation";
 
+import pLimit from 'p-limit';
+
+interface StoreGetOptions {
```

Comment:
Right now it's not so relevant, but I think `Set` will benefit from the same optimization.
Let's leave it for a future PR

Reply:
Ok. I'll leave a `TODO`